code/modules/food&drinks/food/customizables.dm

```diff
@@ -300,6 +300,7 @@
 	icon = 'icons/obj/food/soupsalad.dmi'
 	icon_state = "bowl"
 	flags = OPENCONTAINER
+	materials = list()
```

Comment:
Glass clay bowls that aren't glass, even better!

Hheheh, thanks for clearing it up.